src/analog/commands/uplink/Day.ts

```diff
@@ -97,7 +97,7 @@ class Day extends Command {
         buffer.setUint8(CommandBinaryBuffer.setMagneticInfluenceBit(byte, isMagneticInfluence));
         buffer.setLegacyCounterValue(value);
 
-        return Command.toBytes(COMMAND_ID, buffer.toUint8Array(), true);
+        return Command.toBytes(COMMAND_ID, buffer.toUint8Array(), false);
```

Comment:
isLegacyHeader - flag added due to my misunderstanding and should be removed completely

Reply:
removed